.gitignore

```diff
@@ -5,6 +5,7 @@ outputs/
 test_files/
 config.json
 *.pem
+web/
```

Comment:
Why are we .gitignoring the web directory?  Removing it should be sufficient.

Reply:
I have the web directory in my local project to keep if ever needed in the future

Reply:
Make a branch for that - shouldn't go into the code.